Lesson8.ipynb

```diff
@@ -55,9 +86,68 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# list dataframe in tabular format\n",
+    "# Do this again, this time as a dictionary\n",
+    "fruits = pd.Series({'a': \"Apple\", 'b': \"Banana\", 'm': \"Mango\"})\n",
+    "fruits['m']  # returns 'Mango'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pandas Data structure - DataFrame\n",
     "\n",
-    "df"
+    "A DataFrame is a **2-dimensional** table with labeled axes. It acts like a dict-like container for Series objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's build a DataFrame of fruit inventory across all floors\n",
+    "fruit_inventory = {\n",
+    "    \"fruit\": [\"Apple\", \"Banana\", \"Mango\"],\n",
+    "    \"fruit_count_2F\": [75, 150, 250],\n",
+    "    \"fruit_count_6F\": [80, 120, 150],\n",
+    "    \"fruit_count_8F\": [50, 100, 200],\n",
+    "    \"fruit_count_9F\": [100, 200, 350],    \n",
+    "  }\n",
+    "df1 = pd.DataFrame(fruit_inventory)\n",
+    "df1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# You get a Series if you access a DataFrame's index\n",
+    "df1.fruits  # you can also use df1[\"fruits\"]"
+   ]
```

Comment:
When i run this cell, i got the following error:
`AttributeError: 'DataFrame' object has no attribute 'fruits'`
You need to remove the 's' from 'fruits'